extension/tpch/dbgen/answers/sf100/q01.csv

```diff
@@ -0,0 +1,5 @@
+l_returnflag|l_linestatus|sum_qty|sum_base_price|sum_disc_price|sum_charge|avg_qty|avg_price|avg_disc|count_order
```

Comment:
👋  hi. Where did you get these expected answers? Are they from DuckDB itself or did you generate / compute them somewhere else?

Context is we would like to offer something similar in a Rust generator, and it isn't clear to us how to compute the expected answers for different scale factors:
- https://github.com/clflushopt/tpchgen-rs/issues/103

Reply:
I think we computed them ourselves and then double checked the results by comparing with other db systems (e.g. postgres)

Reply:
Thank you @Tmonster 🙏